Lexdao-Constitutionv02.md

```diff
@@ -0,0 +1,160 @@
+
+LexDAO members, or users of any LexDAO resources, should contact their attorney to obtain advice with respect to any particular legal matter. No reader, user, or browser of LexDAO websites or services should act or refrain from acting on the basis of information on this site without first seeking legal advice from counsel in the relevant jurisdiction. Only your individual personal attorney can provide assurances that the information contained herein – and your interpretation of it – is applicable or appropriate to your particular situation. Use of, and access to, LexDAO, any LexDAO website or any of the links or resources contained within the site do not create an attorney-client relationship between the reader, user, or browser and website authors, contributors, contributing law firms, or committee members and their respective employers.
+
+**Section 1.4 - Cooperative principles.** LexDAO shall be operated in the spirit of cooperative principles framed by the 1995 General Assembly of the International Co-operative Alliance, including: (i) voluntary and open membership; (ii) democratic member control through distributed ledger smart contract voting management; (iii) member economic participation; (iv) autonomy and independence of LexDAO; (v) providing education, training and information; (vi) cooperation with other DAO cooperatives; and (vii) concern for community.
+
+**Section 1.5 - Nondiscrimination.** LexDAO shall not arbitrarily discriminate on the basis of race, nationality, religion, age, gender, sexual orientation, disability, political affiliation, or otherwise.  
+
+**Section 1.6 - Decentralized Autonomous Organization (DAO).** LexDAO is a DAO, an unincorporated association of individuals, entities, associations and/or other persons or groups of persons who all agree to join together in a common purpose. For LexDAO that purpose is encapsulated in our Mission Statement.
+
+# Article II - Membership
+
+**Section 2.1 - Eligibility** Membership as defined in Wyo. Stat. Title 17, Ch. 22 shall be voluntary and application open to any individual of at least 18 years of age whose purpose or presumed intent is to contribute, to build, and to use the services of LexDAO and is willing to accept the responsibilities and terms of membership. 
+
+**Section 2.2 - Admission.** The membership of LexDAO applicants will be granted or denied by vote upon the submission of required information on a prescribed application form, paying any required application fee, and contributing to LexDAO, if any, all as determined by the membership. The contribution requirement may be waived in whole or in part for persons of low income. LexDAO is a decentralized network of members and may admit or deny for any arbitrary purpose or lack of purpose.
```

Comment:
Would like to better understand the vote process.  This may be described later in the document, but would be good to have a grant/deny outline in the admissions process or a reference to the details in appendix/subsequent section.